OpenRA.Mods.Common/Effects/GravityBomb.cs

```diff
@@ -92,7 +95,7 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 				{
 					var dat = world.Map.DistanceAboveTerrain(pos);
 					var shadowPos = pos - new WVec(0, 0, dat.Length);
-					foreach (var r in anim.Render(shadowPos, wr.Palette("shadow")))
+					foreach (var r in anim.Render(shadowPos, wr.Palette(info.ShadowPalette)))
```

Comment:
Looks like you need a null-check here too, otherwise you're just shifting the anim NRE later.